app/filters.php

```diff
@@ -72,10 +72,14 @@
         $comments_template
     );
 
+    $data = collect(get_body_class())->reduce(function ($data, $class) use ($template) {
```

Comment:
I think `$template` is undefined in this scope. Should be `$comments_template`?

Reply:
@mmirus You are right, it's fixed.